bin.src/make_remote_butler.py

```diff
@@ -90,12 +91,43 @@ def _add_chains(butler):
                                      butler.registry)
     defaults = instrument.makeCollectionName("defaults")
     butler.registry.registerCollection(defaults, type=CollectionType.CHAINED)
+    calib_collection = instrument.makeCalibrationCollectionName()
+    butler.registry.registerCollection(calib_collection, type=CollectionType.CHAINED)
     butler.registry.setCollectionChain(
         defaults,
-        [instrument.makeCalibrationCollectionName(), "templates", "skymaps", "refcats"]
+        [calib_collection, "templates", "skymaps", "refcats"]
     )
 
 
+def _hsc_rc2(butler):
+    """fix up some specifics of the HSC-RC2 dataset export
+
+    Parameters
+    ----------
+    butler: `lsst.daf.butler.Butler`
+        The source Butler from which datasets are exported
+    """
+    # Chain calibration collections
+    instrument = Instrument.fromName("HSC", butler.registry)
+    butler.registry.setCollectionChain(
+        instrument.makeCalibrationCollectionName(),
+        [
+            "HSC/calib/DM-32378",
+            "HSC/calib/gen2/20180117",
+            "HSC/calib/DM-28636",
+            "HSC/calib/gen2/20180117/unbounded",
+            "HSC/calib/DM-28636/unbounded",
```

Comment:
> Can the other specific collections be replaced with a query for calibration collections that start with HSC/calib/?

Not really, because there are two other CALIBRATION collections in `/repo/main` with that prefix, and including them would mean including data that aren't needed.

Reply:
You are right. I don't really need the "unbounded" RUN collections. Datasets in the CALIBRATION collections have pointers to other RUN collections.

Reply:
From [this slack thread](https://lsstc.slack.com/archives/C01FBUGM2CV/p1673384221142219)  it looks to me that  https://github.com/lsst-dm/prompt_prototype/blob/main/python/activator/middleware_interface.py#L168 is the right way to go for the moment. To be consistent I'm adding the chain to the new repo (https://github.com/lsst-dm/prompt_prototype/pull/44/commits/7a817edc37ebb2c1c302641e125e81e3e47ce8ed )

It also means I'll make prompt processing's central repo more similar to /repo/main , and less similar to ap_verify_ci_cosmos_pdr2.